src/actions/albums.js

```diff
@@ -0,0 +1,108 @@
+/**
+  Albums related features
+**/
+
+import cozy from 'cozy-client-js'
```

Comment:
As long as `cozy-client-js` is now exposed directly by the stack, we don't want in packed in the final app build. So we should remove this line, add a `/* global cozy */` header, and prefer to use `cozy.client.<method>` in our code (see https://github.com/cozy/cozy-files-v3/pull/110, I'll do the same for photos-v3 :))